<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WebSite</class>
 <widget class="QDialog" name="WebSite">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>963</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WebsiteMPSI</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QTabWidget" name="tabs">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabVieclasse">
      <attribute name="title">
       <string>Vie de classe</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QWidget" name="gridVieClasse" native="true">
         <layout class="QGridLayout" name="formVieClasse"/>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabDS">
      <attribute name="title">
       <string>DS</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QWidget" name="gridDS" native="true">
         <layout class="QGridLayout" name="formDS"/>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabDM">
      <attribute name="title">
       <string>DM</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QWidget" name="gridDM" native="true">
         <layout class="QGridLayout" name="formDM" rowminimumheight="0" columnminimumwidth="0">
          <property name="sizeConstraint">
           <enum>QLayout::SetNoConstraint</enum>
          </property>
          <property name="spacing">
           <number>6</number>
          </property>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabColles">
      <attribute name="title">
       <string>Colles</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QWidget" name="gridColles" native="true">
         <layout class="QGridLayout" name="formColles"/>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabADS">
      <attribute name="title">
       <string>ADS</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QWidget" name="gridADS" native="true">
         <layout class="QGridLayout" name="formADS"/>
        </widget>
       </item>
       <item alignment="Qt::AlignRight|Qt::AlignTop">
        <widget class="QWidget" name="ADSButtons" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QVBoxLayout" name="buttonsADS">
          <property name="spacing">
           <number>6</number>
          </property>
          <item alignment="Qt::AlignRight|Qt::AlignTop">
           <widget class="QPushButton" name="addADSButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignRight|Qt::AlignTop">
           <widget class="QPushButton" name="removeADSButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabCours">
      <attribute name="title">
       <string>Cours</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QWidget" name="gridCours" native="true">
         <layout class="QGridLayout" name="formCours"/>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabInterros">
      <attribute name="title">
       <string>Interros</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QWidget" name="gridInterros" native="true">
         <layout class="QGridLayout" name="formInterros"/>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabTD">
      <attribute name="title">
       <string>TD</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QWidget" name="gridTD" native="true">
         <layout class="QGridLayout" name="formTD"/>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabInfo">
      <attribute name="title">
       <string>Info</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QWidget" name="gridInfo" native="true">
         <layout class="QGridLayout" name="formInfo"/>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabSlidesInfo">
      <attribute name="title">
       <string>SlidesInfo</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QWidget" name="gridSlidesInfo" native="true">
         <layout class="QGridLayout" name="formSlidesInfo"/>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabTDInfo">
      <attribute name="title">
       <string>TDInfo</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <widget class="QWidget" name="gridTDInfo" native="true">
         <layout class="QGridLayout" name="formTDInfo"/>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabTPInfo">
      <attribute name="title">
       <string>TPInfo</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_13">
       <item>
        <widget class="QWidget" name="gridTPInfo" native="true">
         <layout class="QGridLayout" name="formTPInfo"/>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabDSInfo">
      <attribute name="title">
       <string>DSInfo</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_14">
       <item>
        <widget class="QWidget" name="gridDSInfo" native="true">
         <layout class="QGridLayout" name="formDSInfo"/>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabFormulaires">
      <attribute name="title">
       <string>Formulaires</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_15">
       <item>
        <widget class="QWidget" name="gridFormulaires" native="true">
         <layout class="QGridLayout" name="formFormulaires"/>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabAnimations">
      <attribute name="title">
       <string>Animations</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QWidget" name="gridAnimations" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QGridLayout" name="formAnimations">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
         </layout>
        </widget>
       </item>
       <item alignment="Qt::AlignRight|Qt::AlignTop">
        <widget class="QGroupBox" name="AnimationsButtons">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QVBoxLayout" name="buttonsAnimations">
          <property name="spacing">
           <number>6</number>
          </property>
          <item alignment="Qt::AlignRight|Qt::AlignTop">
           <widget class="QPushButton" name="addAnimationsButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignRight|Qt::AlignTop">
           <widget class="QPushButton" name="removeAnimationsButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="transferButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
